readthedocs/templates/docsitalia/overrides/projects/onboard_import.html

```diff
@@ -4,7 +4,7 @@
 <div class="onboard onboard-import-project">
   <h2>{% trans "Ready to share your documentation" %}{% if request.user.first_name %}, {{ request.user.first_name }}{% endif %}?</h2>
 
-  {% with getting_started_url="http://docs.readthedocs.io/en/latest/getting_started.html" %}
+  {% with getting_started_url="http://guida-docs-italia.readthedocs.io/it/latest/" %}
```

Comment:
Sorry to be picky: can you use `https://` here? Thanks!

Reply:
Sure, I c&p'ed it from our about page though :)

Reply:
Done and fixes urls in about page too